### ISSUE TITLE
How to change subtitle color?

### ISSUE THREAD

**Issue by SilverMarcs** [iina/iina#3934]
When the color picker pops up, how do I apply the color I selected? There is no option to apply the selected color. Pressing Enter doesnt do it. Upon closng the color picker popup, the subtitle color remains the same as before (white).

**Comment by low-batt** [comment]
Click on a color in the color picker. That choice should immedately result in the associated subtitle color changing in the IINA preferences as seen here:
![issue-3934](https://user-images.githubusercontent.com/86170219/188284795-1d43e494-b11a-4373-922f-f44b64df347d.jpg)

This example was created using IINA 1.3.0 under macOS 12.5.1.

If that is not working then please provide more details.

By the way, perfectly fine that this was entered as an issue, however GitHub would have us use the new [Discussions Q&A](https://github.com/iina/iina/discussions/categories/q-a) feature for questions.

**Comment by SilverMarcs** [comment]
It seems to work as intended when setting from the preferences menu. To be more specific, the issue here is when trying to change from the video player itself after navigating to Subtitles tab (clicking 3 dot menu on the player controls). If it was never intended to be able to change font color from this page, perhaps restrict the button from popping up the color picker altogether?

**Comment by low-batt** [comment]
The color controls on the `Subtitles` tab of the `Quick Setting Panel` should behave the same way as the controls in in preferences.

I just tried this under macOS 12.5.1 on my MacBook Pro M1 machine. I brought up the color picker using the `Quick Setting Panel`. When I picked a color the color displayed in the panel was updated. The subtitle being displayed also changed color.

So the problem did not reproduce for me. Possibly there is a subtle defect lurking in this area that is dependent upon the environment in some way. I'm at a loss as to what that could be. What version of macOS are you running under?

As you indicated it is working from preferences and not from the `Quick Setting Panel` it sounds like the video does not have burned in subtitles. Subtitle style controls obviously do not work with image-based subtitles.
